pkg/ebpf/flow_filter.go

```diff
@@ -163,43 +172,67 @@ func (f *Filter) getFilterValue(config *FilterConfig) (BpfFilterValueT, error) {
 	return val, nil
 }
 
-func getSrcPorts(config *FilterConfig) (uint16, uint16) {
+func getSrcPortsRange(config *FilterConfig) (uint16, uint16) {
 	if config.FilterSourcePort.Type == intstr.Int {
 		return uint16(config.FilterSourcePort.IntVal), 0
 	}
-	start, end, err := getPortsFromString(config.FilterSourcePort.String())
+	start, end, err := getPortsFromString(config.FilterSourcePort.String(), "-")
 	if err != nil {
 		return 0, 0
 	}
 	return start, end
 }
 
-func getDstPorts(config *FilterConfig) (uint16, uint16) {
+func getSrcPorts(config *FilterConfig) (uint16, uint16) {
+	port1, port2, err := getPortsFromString(config.FilterSourcePort.String(), ",")
+	if err != nil {
+		return 0, 0
+	}
+	return port1, port2
+}
```

Comment:
To support more than two ports, instead of port1 and port2, return a port array.  Then it just needs a C function in flow\_filter.h to check whether a port is in the array or not.  The 'if' statement will even be shorter.

Reply:
the prob is not about arry of not the main prob is we configure the agent via envvar and we cant use array there unless we do something else instead of envvar

Reply:
Why wouldn't you be able to set the environment variable, FILTER_SOURCE_PORTS, to contain more than two ports like "53,5353,80,8000" or any number of individual ports?

Reply:
we can't use variable sized arrays so we have to have known size how big that array will be

Reply:
okay, that's quite limiting.